pkg/jobstore/boltdb/store.go

```diff
@@ -1028,6 +1056,35 @@ func (b *BoltJobStore) appendExecutionHistory(tx *bolt.Tx, updated models.Execut
 	return nil
 }
 
+func (b *BoltJobStore) appendEvaluationHistory(tx *bolt.Tx, eval models.Evaluation) error {
+	historyEntry := models.JobHistory{
+		Type:             models.JobHistoryTypeEvaluationLevel,
+		JobID:            eval.JobID,
+		EvaluationID:     eval.ID,
+		EvaluationStatus: eval.Status,
+		Comment:          eval.Comment,
+		Time:             time.Unix(0, eval.ModifyTime),
+	}
```

Comment:
It may also be valuable to also include the `TriggeredBy` field in this entry from the evaluation, but I have opted not to as it will further complicate the fields of `JobHistory`